### PR TITLE
[DBIO-1835 & DBIO-1837] feat : Genetic Analysis Order Listener : Rejected, Refunded, ResultReady, and Fulfilled

### PR DESCRIPTION
**Pull request by JackyRahman**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-1835](https://blocksphere2020.atlassian.net/browse/DBIO-1835)
[https://blocksphere2020.atlassian.net/browse/DBIO-1837](https://blocksphere2020.atlassian.net/browse/DBIO-1837)

### Changelog / Description
- [feat: create model and listener for genetic analysis rejected](https://github.com/debionetwork/debio-backend/commit/a3fe178bcfc7121da709f37d583520865a5ebcf9)
- [feat: create GA order refunded listener](https://github.com/debionetwork/debio-backend/commit/fedf63953ae739e765baf698d23b3079e7faa8b4)
- [tests: create unit test GA order refunded listener](https://github.com/debionetwork/debio-backend/commit/cb80241134f108768c45b1eac6b035d9d36d0ffb)
- [tests: create unit test for GA rejected](https://github.com/debionetwork/debio-backend/commit/bccc318bdf116f37e029d66dda63c754786bdef8)
- [feat: create listener GA result ready](https://github.com/debionetwork/debio-backend/pull/268/commits/1e55e2436ed4544904da45c1f0b788e204e6da7a)
- [feat: create listener GA fulfilled](https://github.com/debionetwork/debio-backend/pull/268/commits/2d39b3e965ca33ea81ccdffd55260e8645c5646e)
- [tests: create unit test for GA order fulfilled and result Ready](https://github.com/debionetwork/debio-backend/pull/268/commits/a48258d02453791249dfbe2c9c97c65e7010f368)

### CODE REVIEW

File: src/common/polkadot-provider/models/genetic-analisys/index.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+  
+  genetic_analysis_tracking_id: string;
+  genetic_analystId: string;
+  owner_id: string;
+  report_link: string;
+  comment: string;
+  rejected_title: string;
+  rejected_description: string;
+  genetic_analysis_orderId: string;
```
Comment (hilyds) [inline src/common/polkadot-provider/models/genetic-analisys/index.ts:26]: use snake case for `id` in `genetic_analystId` and `genetic_analysis_orderId`

File: src/common/polkadot-provider/models/genetic-analisys/index.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { convertSubstrateNumberToNumber } from '../../';
+import { GeneticAnalysisStatus } from './genetic-analysis-status'
+
+export class GeneticAnalysis {
+  constructor(anyJson: any) {
+    this.genetic_analysis_tracking_id = anyJson.geneticAnalysisTrackingId;
+    this.genetic_analystId = anyJson.geneticAnalystId;
```
Comment (hilyds) [inline src/common/polkadot-provider/models/genetic-analisys/index.ts:7]: use snake case for `id` in `genetic_analystId` and `genetic_analysis_orderId`

Comment (hilyds) [reply src/common/polkadot-provider/models/genetic-analisys/index.ts:7]: also typo for the file name

File: src/common/polkadot-provider/models/genetic-analisys/index.ts (hunk 1 above)
Comment (JackyRahman) [reply src/common/polkadot-provider/models/genetic-analisys/index.ts]: done

File: src/common/polkadot-provider/models/genetic-analisys/index.ts (hunk 2 above)
Comment (JackyRahman) [reply src/common/polkadot-provider/models/genetic-analisys/index.ts]: done

File: src/common/polkadot-provider/models/index.ts [hunk 3]
```diff
@@ -6,3 +6,4 @@ export * from './genetic-testing';
 export * from './service-request';
 export * from './genetic-analysts';
 export * from './genetic-analysis-orders';
+export * from './genetic-analisys';
```
[the remark this answers is not in the document]
Comment (JackyRahman) [reply src/common/polkadot-provider/models/index.ts]: done

**Review by hilyds — APPROVED** [review]
Looks good 👍

**Pull request merged**